app/scripts/modules/core/pipeline/config/stages/webhook/webhookStage.ts

```diff
@@ -16,15 +17,17 @@ interface ICommand {
 
 export class WebhookStage {
   static get $inject() {
-    return ['stage', 'jsonUtilityService'];
+    return ['stage', 'jsonUtilityService', '$uibModal'];
   }
 
   public command: ICommand;
   public viewState: IViewState;
   public methods: string[];
 
   constructor(public stage: any,
-              private jsonUtilityService: JsonUtilityService) {
+              private jsonUtilityService: JsonUtilityService,
+              private $uibModal: IModalService
+  ) {
```

Comment:
Move this to the line above, i.e. `private $uibModal: IModalService) {`

(other people might disagree; that is maybe just my preference and not enforced by our linter)